URBNSwiftAlert/Classes/AlertAction.swift

```diff
@@ -0,0 +1,75 @@
+//
+//  AlertAction.swift
+//  Pods
+//
+//  Created by Kevin Taniguchi on 5/23/17.
+//
+//
+import Foundation
+
+public class AlertAction: NSObject {
+    public enum ActionType {
+        case normal, destructive, cancel, passive, custom
+    }
+    
+    let type: ActionType
+    let shouldDismiss: Bool
+    let isEnabled: Bool
+    let completion: ((AlertAction) -> Void)?
+    public var button: UIButton?
+    var title: String?
+    
+    var isButton: Bool {
```

Comment:
`hasButton`? Also, I don’t see this being used anywhere.

Reply:
removed